R/add_dockerfiles_renv.R

```diff
@@ -110,7 +110,7 @@ add_dockerfile_with_renv_ <- function(
       )
     ) {
       out <- pkgbuild::build(
-        path = ".",
+        path = get_golem_wd(),
```

Comment:
I think here we should rely on passing arg from the parent function, so that if source_folder is changed, it's passed to pkgbuild::build().

Reply:
done. 

let me know about any changes in the bootstrap_type functions (see PR's initial comment)   @ColinFay  

these functions are used internally only, taking arguments from their respective parent functions. still might be useful to have `get_golem_wd` defaults instead of "."